src/algebra/category/Group/injective.lean

```diff
@@ -106,4 +106,145 @@ module.Baer.injective $
       congr, }, },
 end
 
+namespace divisible_emb
+
+-- In this section we prove that any additive abelian group A can be embed into a divisible group,
+-- namely `∏ᵢ (ℚ⧸ℤ)` where `i` indexing over all morphism `A →+ ℚ⧸ℤ`
+
+/-- integers as a subgroup of ℚ -/
+protected def ℤ_as_ℚ_subgroup : add_subgroup ℚ :=
```

Comment:
I would call this `rat.int_subgroup` (in the root namespace).

Reply:
Also there should be a `ℤ →+ ℚ` map somewhere in mathlib, and this is just the range of it.